docs/docs/users/general/creating-campaigns.md

```diff
@@ -0,0 +1,42 @@
+---
+id: creating-campaigns
+title: Creating and managing campaigns
+---
+Here is more information about campaigns. These can be either petition campaigns, event campaigns or fundraising campaigns. 
+
+## General info
+![](https://s3-eu-west-1.amazonaws.com/bureaubolster/Wings_2019-03-25_11-31-17.png)
+
+At Campaigns, you can manage 1) Events or 2) Petitions. You can 3) click on existing campaign to edit it, or 4) Create a new one. 
+
+Events are geared towards getting people to visit an event; petitions are aimed towards letting people sign a policy proposal, letter of support or some other kind of petition. 
+
+## Where is your campaign page?
+Campaigns get their own page on your website by default; by defining the 'slug', you define its url.  Every type of campaign has its own directory, though; if your slug is 'demo-fundraiser' and your website url is mywebsite.com, the url of this specific fundraiser page can be found at 'mywebsite.com/fundraisers/demo-fundraiser'. 
```

Comment:
Campaigns don't necessarily get their own page. In HB they do, but content management doesn't have any direct result on a website by default.

Reply:
Changed the wording to specify that they most often do. HB is Wings core, almost all projects will have this, and this information needs to be in the manual to let users find where their campaign can be found. Campaigns get a 'slug' for a reason. An indication that some projects will not give campaigns their own page is enough; it is a core feature of Wings  that they do get their own page.